### PR TITLE
add Saarfahrplan profile

### PR DESCRIPTION
**Pull request by derhuerst**

### CODE REVIEW

File: p/saarfahrplan/example.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+'use strict'
+
+const createClient = require('../..')
+const oebbProfile = require('.')
```
Comment (juliuste) [inline p/saarfahrplan/example.js]: We should rename this

File: p/saarfahrplan/example.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+const client = createClient(oebbProfile, 'hafas-client-example')
```
Comment (juliuste) [inline p/saarfahrplan/example.js]: same here

**Comment by derhuerst** [comment]
Only unrelated tests are failing.

**Comment by juliuste** [comment]
By the way: We might consider using something like your `tap-min` module in the `npm test` script given the output size we have reached now.

**Comment by derhuerst** [comment]
I've purposefully avoided `tap-min`, because it swallows/shortens TAP output too much. Maybe there's something like [`tap-spec`](https://github.com/scottcorgan/tap-spec#readme) that doesn't print successful assertions.

**Comment by derhuerst** [comment]
@ski7777 Did you see this? I think you had asked for SaarVV support a while ago.

**Pull request merged**